.circleci/config.yml

```diff
@@ -204,7 +204,6 @@ commands:
               --var SMTP_SECURE=${<< parameters.smtp_secure >>} \
               --var SMTP_IGNORE_TLS=${<< parameters.smtp_ignore_tls >>} \
               --var FROM_EMAIL_ADDRESS=${<< parameters.from_email_address >>}
-              --var SUPPRESS_ERROR_LOGGING=${<< parameters.suppress_error_logging >>}
```

Comment:
Keep this line and add the backslash to the proceeding line at the end

Reply:
I saw the syntax error earlier, fixing that yields this: https://app.circleci.com/pipelines/github/HHS/Head-Start-TTADP/12536/workflows/31760245-6ce2-495f-a22c-d0f86b3565ad/jobs/70422

Reply:
```
Incorrect Usage: invalid argument for flag `--var' (expected []template.VarKV): Expected var 'SUPPRESS_ERROR_LOGGING=' to specify non-empty value
```

Reply:
I can add it to cloud.gov first to bypass that error but I'm still not sure why we'd need to add it to this particular command

Reply:
I mean Circle Ci, not Cloud.gov

Regardess, if you want to take ownership of this and tweak it a different way, I'll hold off merging until you can properly start your work day. Turns out we are waiting on demos to deploy to Prod regardless

Reply:
Even though this is a non-secret variable, it will be easier to turn it on/off if we have it in CircleCI

Reply:
Does it need to be passed to cf deploy then, is that what you mean?